viadot/sources/salesforce.py

```diff
@@ -173,6 +193,18 @@ def bulk_upsert(
     def download(
         self, query: str = None, table: str = None, columns: List[str] = None
     ) -> List[OrderedDict]:
+        """
+        Dowload all data from the indicated table or the result of the specified query.
```

Comment:
typo

Reply:
[✅📝Improved docstings in Salesforce class](https://github.com/dyvenia/viadot/pull/629/commits/570b5797c61f1e1ca58ec24b03d4d67b96d66706)